src/ui/components/Markdown/Markdown.tsx

```diff
@@ -0,0 +1,23 @@
+import React, { FunctionComponent } from 'react'
+
+import { parseMarkdown } from 'features/offer/helpers/parseMarkdown/parseMarkdown'
+import { MarkdownPartProps } from 'features/offer/types'
+import { MarkdownPart } from 'ui/components/MarkdownPart/MarkdownPart'
+
+type Props = {
+  text: string
```

Comment:
en react, généralement, quand on veut donner le contenu qui sera affiché dans un composant, on le passe entre les balises (`<Balise>le contenu</Balise>`), donc dans `children`

est-ce qu'il y a des raisons pour avoir choisi `text` et pas `children` ?

Reply:
La fonctionne `parseMarkdown` s'attend à avoir un texte. Tu transformerais le `ReactNode` en `string` avant d'appeler la fonction ?

Reply:
est-ce qu'on peut contraindre le `children` pour toujours avoir une `string` ?

```suggestion
  children: string
```

Reply:
Ah yes ça le fait ! je te laisse jeter un oeil au dernier commit si tu veux